src/component/startGame/index.js

```diff
@@ -0,0 +1,20 @@
+import React,{Component} from "react";
+
+class StartGame extends Component {
+    state={
+        name:""
+    }
+    onChange=(event)=>{
```

Comment:
How the name will be passed for other components ?

Reply:
we solved it on other laptop and you will see it in the next pull request
@ghassanmas